### ISSUE TITLE
Accessibility: Plugin favorites heart color contrast failure

### ISSUE THREAD

**Issue by amberhinds** [WordPress/wordpress.org#265]
On the plugin single page ([Gutenberg plugin test page](https://test.wordpress.org/plugins/gutenberg/)), the plugin favorites heart in its unselected state is #cbcdce on a white background which is 1:59:1.

<img width="205" alt="Screenshot 2024-04-16 at 5 45 36 PM" src="https://github.com/WordPress/wordpress.org/assets/13153456/4ea75e44-01f9-4277-8391-3bf6d41622c7">

Here's the[ contrast on WebAIM's checker.](https://webaim.org/resources/contrastchecker/?fcolor=CBCDCE&bcolor=FFFFFF) This fails for UI components and many people with low-vision will not even know that is there.

Because we don't want to fail Use of Color, it would not be sufficient to just darken the color of the heart. I'd recommend adding a border to it so it has a border and light or no color if unselected and a solid color if hovered or selected.

**Comment by StevenDufresne** [comment]
@dd32 Mentioned we could use new "star" icon used [elsewhere](https://wordpress.org/patterns/?new-theme=1) in a previous ticket which makes sense.



![Image](https://github.com/WordPress/wordpress.org/assets/1657336/5fb28e69-1f89-45f3-90ac-51eaea61e99d)

**Comment by ndiego** [comment]
I like the star, and it makes "favoriting" things consistent with the new Patterns design. 

While we should definitely fix this issue asap, I don't think it should block launching the new design since the color is the same as it is on the current theme.

**Comment by StevenDufresne** [comment]
I wonder if it would be mistaken with plugin reviews?

<img width="193" alt="Screenshot 2024-04-17 at 2 26 33 PM" src="https://github.com/WordPress/wordpress.org/assets/1657336/264b5570-1a6f-4551-9edc-9340f6c91a09">

**Comment by ndiego** [comment]
> I wonder if it would be mistaken with plugin reviews?

That's true, but I think "staring" something is also a common way to favorite things. I am thinking about Slack and Figma.

![staring](https://github.com/WordPress/wordpress.org/assets/4832319/47f841ee-c1c8-49c2-afdc-a30a204f0e2a)

Would it be possible to add a tooltip to the icon (like in the gif) that provides more context about what it is? Even with the current heart icon, it's not 100% clear what it does if you don't associate a heart with favoriting a plugin. 

Thoughts @WordPress/meta-design?

**Comment by jasmussen** [comment]
Yep, good to have a tooltip.

**Comment by ndiego** [comment]
Also, we could keep this star grey/black like it is in the [Patterns refresh](https://wordpress.org/patterns/?new-theme=1) so it's visually different than the pomegranate star ratings for the plugins.

**Comment by amberhinds** [comment]
RE the tool tip, I'll admit that I don't even know what this does, really.
Assumingly there's a way or a place to view all your favorite plugins, but
I have no idea what that is or how to do it.

It might be helpful to make it more clear where to view favorites.

**Comment by ndiego** [comment]
> It might be helpful to make it more clear where to view favorites.

In the new design, there is a link in the local navigation. 

<img width="571" alt="image" src="https://github.com/WordPress/wordpress.org/assets/4832319/77e34b05-4f1d-4db2-b937-9670fdf36bf6">

**Comment by ryelle** [comment]
There's a page in wp-admin where you can view a user's favorites (presumably yours, but there's no connection between wp.org and assorted WordPress sites). They also show up [on your profile](https://profiles.wordpress.org/ryelle/#content-favorites).

<img width="728" alt="Add new plugins, favorites tab" src="https://github.com/WordPress/wordpress.org/assets/541093/f25cdc0b-5536-4558-8b2b-84a1408895bb">

Maybe a little copy could be added to [the favorites page](https://test.wordpress.org/plugins/browse/favorites/), linking off to some documentation (like https://wordpress.org/documentation/article/plugins-add-new-screen/)? (Though this is getting into separate issue territory 🙂 )

**Comment by amberhinds** [comment]
I have a separate issue that I opened for larger functionality issues on this button beyond color contrast: https://github.com/WordPress/wordpress.org/issues/266

It probably makes sense for this to be part of that discussion, because the role needs to be changed anyway. I will add some additional thoughts there and link back to these comments.

**Comment by fcoveram** [comment]
I think we’re mixing three different concepts here. The actions I have noticed on the web when browsing and interacting with directories are:

- **Liking:** A +1 action to note preference. In many cases it influences the recommendation algorithm.
- **Saving:** Adding something to one or several collections. Those can be public or private.
- **Rating:** Scoring based on a scale. Sometimes, the evaluation is public and users can see their scorings.

The last two are currently on the WordPress site, unless I’m missing something. If we are using stars for rating, then we can not use it for saving. It would quickly lead to confusion.

There is no standard icon for saving, it swings between a heart and a bookmark. Therefore, the safest way to reduce misinterpretations is by adding a label next to it. In that vein, and following @ndiego's example, Figma also uses heart and bookmark for liking and saving, respectively.

<img src="https://github.com/WordPress/wordpress.org/assets/895819/b350dd66-c22d-4bba-9d79-803b2ae43495" alt="Screenshot of actions in Figma Community" width=300>

Regarding preferences, I’m drawn to using a bookmark icon. The heart feels like an overrepresentation of the action as “love” is also conveyed with the heart.

**Comment by ryelle** [comment]
The favorite action now uses the same component as Themes & Patterns, so the heart is inside a button, and is #656a71, which has a 5.4:1 contrast. It uses a different icon for favorited/unfavorited state, so the heart is filled if the if the current plugin/theme/pattern is favorited, and empty if not.

See #350 for details & some before/after screenshots.

**Comment by fcoveram** [comment]
It looks great. The loading animation was a nice solution 👏

**Comment by ndiego** [comment]
Yeah, looks great! 👌
